default.nix

```diff
@@ -4,14 +4,45 @@ with import (builtins.fetchGit {
   rev = "95fed28ac372c61eb83c87ad97c24b0f957827bf";
 }) {};
 
+let
+  build =
+    writeShellScriptBin "postgrest-docs-build"
+      ''
+        set -euo pipefail
+
+        sphinx-build -b html -a -n . _build
```

Comment:
This is not going to work outside nix-shell (e.g. when using `nix-env -iA`), as sphinx-build is not taken from nixpkgs. You need to do something like `${pythonPackages.sphinx}/bin/sphinx-build ...`

Reply:
Ah, thanks for the explanation. I was kind of confused why it **did** work, because in your scripts you always prefix that, and it kind of made sense... - but I only tried inside nix-shell.